docs/ballerina-by-example/examples/grpc-bidirectional-streaming/grpc-bidirectional-streaming.bal

```diff
@@ -14,12 +14,12 @@ endpoint grpc:Service ep {
     serverStreaming:true,
     generateClientConnector:false}
 service<grpc:Endpoint> chat bind ep {
-    map consMap = {};
-    onOpen (endpoint client) {
+map consMap = {};
```

Comment:
I am not sure this will work now. Can you change to map consMap; ? "= {}" will not work I think. 

Also there seems to be a formatting issue.